routes/web.php

```diff
@@ -37,24 +37,29 @@
     Route::post('/interactivity/floor', 'InteractiveController@floor')->name('floorInteractivity');
 
     Route::post('photo', 'PhotoController@store')->name('photo.store');
-    Route::post('photo/parcel', 'PhotoController@parcelStore')->name('photo.parcelStore');
+    Route::post('photo/parcel/{project}', 'PhotoController@parcelStore')->name('photo.parcelStore');
```

Comment:
After selecting the project, actually we dont need to send project to anywhere because I set project_id to session. Also this url should change to numbering.